KomaMRICore/test/runtests.jl

```diff
@@ -161,18 +153,79 @@ end
 end
 
 @testitem "Phantom" tags=[:core] begin
+
```

Comment:
This type of test will probably break if we add more properties to the Phantom object.

Reply:
Edited to receive specific keyvars.